tests/OmniSharp.Roslyn.CSharp.Tests/ImplementTypeFacts.cs

```diff
@@ -120,7 +120,7 @@ private async Task VerifyImplementType(string code, string expectedChange, Dicti
                     Column = point.Offset,
                     FileName = testFile.FileName,
                     Buffer = testFile.Content.Code,
-                    Identifier = "False;False;True:global::IFoo;AssemblyName;Microsoft.CodeAnalysis.ImplementInterface.AbstractImplementInterfaceService+ImplementInterfaceCodeAction;",
+                    Identifier = "False;False;True:global::IFoo;Microsoft.CodeAnalysis.ImplementInterface.AbstractImplementInterfaceService+ImplementInterfaceCodeAction;",
```

Comment:
Assembly Name removed in https://github.com/dotnet/roslyn/commit/9c8c21f7f79d4ef833573f7e251b2039026b20c0